.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2.0
+
+jobs:
+    build:
+        docker:
+            - image: circleci/node:6.10
+        steps:
+            - checkout
+            - restore_cache:
+                key: node-{{ checksum "package.json" }}
+            - run:
+                name: Install dependencies
+                command: npm install
+            - run:
+                name: Run Tests
+                command: npm test
+            - save_cache:
```

Comment:
Surely we should save the cache _before_ running the tests?
(even if cache restoring still runs all the steps, running the tests afterwards still feels more correct to me)